docs/source/stability_policy.rst

```diff
@@ -0,0 +1,132 @@
+Stability Policy
+================
+
+The ``python-telegram-bot`` library (PTB) tries to provides a stable interface to its users.
+As both the user base and the complexity of the library has grown significantly since PTBs conception, we introduced below explicit stability Policy with version NEXT.VERSION.
+
+Large parts of the :mod:`telegram` package are the Python representations of the Telegram Bot API, the stability policy of which PTB can not influence.
+This policy hence includes some special cases for those parts.
+
+What does this policy cover?
+----------------------------
+
+This policy includes any API or behavior that is documented in this documentation.
```

Comment:
Since this is readable outside of the doc website, I would link to it. Maybe

```suggestion
This policy includes any API or behaviour that is covered in PTBs documentation, found at https://docs.python-telegram-bot.org/en/stable/
```

Reply:
Is it? Where? 🤔

Reply:
at least in the github repo :D

But you are right, probably not needed, wasn't sure when suggesting this either.